app/controllers/constructions_controller.rb

```diff
@@ -14,6 +14,7 @@ def index
           title
           payload
           dataProvider {
+            id
```

Comment:
there are a few of these `id` additions, but they are not used currently, or am i wrong? what about not requesting them, if not used?

Reply:
removed